src/site/content/en/blog/introducing-aurora/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Introducing Aurora
+subhead: A collaboration between Chrome and open-source web frameworks & tools
+authors:
+  - shubhie
+  - addyosmani
+  - houssein
+date: 2021-06-15
+hero: image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/KvZQXFKIGKEzAjxzf5bF.jpg
+alt: Night sky
+
+description: This article introduces Aurora, a Chrome initiative to collaborate closely with open-source frameworks
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - aurora
+---
+
+On the Chrome team, we care about user-experience and a thriving web ecosystem.
+We want users to have the best possible experience on the web, not only with
+static documents but also when they use rich, highly-interactive applications.
+
+Open-source tools and frameworks play a large role in enabling developers to
+build modern apps for the web, while also supporting good developer experiences.
+These frameworks and tools empower companies of all sizes, as well as
+individuals building for the web.
+
+We believe that frameworks can also play a big role in helping developers with
+key quality aspects such as performance, accessibility, security, mobile
+readiness. Instead of asking every developer and site owner to become an expert
+in these areas and keep up with the constantly changing best practices, the
+framework can support these with baked-in solutions. This empowers developers
+and enables them to focus on building product features.
+
+In a nutshell, our vision is that a high bar of UX quality becomes a side effect
+of building for the web.
+
+## Aurora: a collaboration between Chrome and open-source web frameworks & tools
+
+For almost two years, we have worked with some of the most popular frameworks
+such as Next.js, Nuxt.js and Angular, working to improve web performance. We've
+also funded popular tools and libraries such as Vue, Eslint, Webpack. Today, we
+are giving this effort a name - **Aurora**.
+
+An aurora is a natural light display that shimmers in the sky. As we are trying
+to help user experiences built with frameworks shimmer and shine, we thought
+this name was an appropriate choice.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/lxaUR7Za1DoQMJuJEZHP.png",
+alt="Aurora logo", width="800", height="450" %}
+
+In the coming months, we'll be sharing a lot more detail on Aurora. This is a
+collaboration between a small team of Chrome engineers (internally codenamed
+WebSDK) with experience working on front-end stacks, and framework and tooling
```

Comment:
this doesn't read right. What about:
"... , and contributing to frameworks and tooling"

Reply:
Yeah I think that still sounds a little confusing, so I simplified it to "between a small team of Chrome engineers and framework authors"